changelog/Microsoft.DirectoryServices.json

```
@@ -145,6 +145,42 @@
       "CreatedDateTime": "2021-12-01T00:00:00.000Z",
       "WorkloadArea": "Identity and Access",
       "SubArea": "Directory management"
+    },
+       {
+      "ChangeList": [
+        {
+          "Id": "0f3be1ec-d352-4eb8-ad76-c8e1e1a615eb",
+          "ApiChange": "Resource",
+          "ChangedApiName": "serviceplanInfo",
+          "ChangeType": "Addition",
+          "Description": "Added **errorStatus** to the to the provisioningStatus property of property of the [servicePlanInfo resource type](https://docs.microsoft.com/graph/api/resources/serviceplaninfo?view=graph-rest-1.0) resource.",
+          "Target": "provisioningStatus"
```

Proposed change because your changes affect the property, not the resource.
```suggestion
          "Id": "0f3be1ec-d352-4eb8-ad76-c8e1e1a615eb",
          "ApiChange": "Property",
          "ChangedApiName": "provisioningStatus",
          "ChangeType": "Addition",
          "Description": "Added **ErrorStatus** to the list of possible values for the **provisioningStatus** property of the [servicePlanInfo](https://docs.microsoft.com/graph/api/resources/serviceplaninfo?view=graph-rest-1.0) resource.",
          "Target": "provisioningStatus"
```